src/main/java/frc/robot/RobotContainer.java

```diff
@@ -178,7 +177,7 @@ private void configureBindings() {
         ));
         operatorControllerRightBumper.whileTrue(
           new ConditionalCommand(
-            new IntakeCube(intake),
+            new IntakeCone(intake),
```

Comment:
This seems like an accident?

Reply:
No, that was a deliberate change. We never use the end of arm cube intake, and it is easier for me to use the same button to intake everything. I would like to keep this change.